prometheus/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+mod bank_metrics;
+mod cluster_metrics;
+mod token;
+mod utils;
+
+use solana_gossip::cluster_info::ClusterInfo;
+use solana_runtime::bank_forks::BankForks;
+use std::{
+    sync::{Arc, RwLock},
+    time::SystemTime,
+};
+
+pub fn render_prometheus(
+    bank_forks: &Arc<RwLock<BankForks>>,
+    cluster_info: &Arc<ClusterInfo>,
+) -> Vec<u8> {
+    let current_bank = bank_forks.read().unwrap().working_bank();
```

Comment:
Is the `bank_forks` the thing where we can get the banks for the different commitment levels from?

Reply:
This is the latest, I couldn't find an API in the struct to get slots at different commitment levels. We can do it at a later time